lib/util/string_extensions.dart

```diff
@@ -0,0 +1,9 @@
+extension StringExtensions on String {
```

Comment:
I'm not sure where this is used but I'd suggest using TextOverflow to trim text in UI: https://api.flutter.dev/flutter/rendering/TextOverflow-class.html

Reply:
This is not for flutter widgets.